.github/scripts/test.sh

```diff
@@ -0,0 +1,2 @@
+flake8 --exit-zero --statistics --config build_tools/flake8.cfg RefRed
+cd test;coverage run peak_finder_derivation_test.py
```

Comment:
I suggest you add package `pytest-cov` to file `requirements_dev.txt` and then substitute this line with
```bash
pytest --cov=RefRed test/
```